### PR TITLE
Encode database ontology and vocabulary

### PR DESCRIPTION
**Pull request by PietroLiuzzo**
https://w3id.org/encode would become the base namespace for the ontology and vocabularies of a starting project Erasmus project called Encode https://site.unibo.it/encode/it.

### CODE REVIEW

**Comment by dgarijo** [comment]
@PietroLiuzzo,
please add who the maintainer for this w3id will be in the readme file. If possible, please also squash your commits.

File: encode/.htaccess [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^ontology$ https://encode.uni-hamburg.de/ontology [R=302,L]
+RewriteRule ^vocabulary$ https://encode.uni-hamburg.de/vocabulary [R=302,L]
+RewriteRule ^modules/?$ https://encode.uni-hamburg.de/modules/ [R=302,L]
+RewriteRule ^vocabulary/?$ https://encode.uni-hamburg.de/competences/$1 [R=302,L]
+RewriteRule ^openings/?$ https://encode.uni-hamburg.de/openings/$1 [R=302,L]
+RewriteRule ^modules/?$ https://encode.uni-hamburg.de/modules/$1 [R=302,L]
+RewriteRule ^competences/(.+)$ https://encode.uni-hamburg.de/competences/$1 [R=302,L]
+RewriteRule ^openings/(.+)$ https://encode.uni-hamburg.de/openings/$1 [R=302,L]
```
Comment (davidlehn) [inline encode/.htaccess:10]: Some of the above look wrong, please check.  There are duplicates and mismatched names and $1 with no patterns.

**Comment by PietroLiuzzo** [comment]
Thansk a lot! I have improved the readme, and, I hope, corrected and simplified the .htaccess . I think I would really do with some advise on how to best achieve this. While for the ontology and vocabulary the redirect is simple, and concepts in there will have hashes, e.g. https://w3id.org/encode/ontology#Competence and  https://w3id.org/encode/vocabulary#Encode, what we would like it to do is that for each of the three classes competences, modules, and openings if https://w3id.org/encode/modules or https://w3id.org/encode/modules/ is requested it will redirect to https://encode.uni-hamburge.de/modules but when a specific module id is requested it goes there, so if https://w3id.org/encode/modules/m1 is requested this is redirected to https://encode.uni-hamburge.de/modules/m1 . I am not sure if  `^modules/?(.+)$` would do this or I need  ^modules/?([a-zA-Z0-9]*)$ where the / may or may not be there and the last part of the pattern is a free combination of any number of letters and numbers to redirect to https://encode.uni-hamburg.de/modules/$1 Thanks again!

**Comment by PietroLiuzzo** [comment]
I have searched on how to squash my commits but I am a bit uncomfortable with how to do that. I am happy to be pointed to clearer instructions which I may have not found of course! However, perhaps can these commits be squashed on merge with the Squash and Merge option, if this will be merged?

**Comment by dgarijo** [comment]
@PietroLiuzzo I have merged it.
If you are not sure about the htaccess, I recommend testing it locally. You can do it by installing a tomcat locally; or using XAMPP (which includes Tomcat and configures it for you).

**Comment by PietroLiuzzo** [comment]
Thanks a lot!

**Comment by dgarijo** [comment]
Note that https://w3id.org/encode has no redirection

**Comment by PietroLiuzzo** [comment]
Do you think it may be a problem? In which case I will add a redirect. What we need to be permanently addressed are ontology, vocabularies and entities.

**Comment by dgarijo** [comment]
Will it be a problem? Not necessarily. But if you expect someone to find something at https://w3id.org/encode, it will be. Up to you :)

**Pull request merged**